make/gen.cmake

```diff
@@ -97,6 +97,8 @@ function(V2X)
   )
 
   add_file_target(FILE "${V2X_NAME}.pb_type.xml" GENERATED)
+  get_file_target(SRC_TARGET_NAME "${V2X_NAME}.pb_type.xml")
+  set_target_properties(${SRC_TARGET_NAME} PROPERTIES INCLUDE_FILES "${INCLUDE_FILES}")
```

Comment:
This should be the pb_type XML include list, not the Verilog include list.

Reply:
yeah, should now be fixed